cloud-storage.go

```diff
@@ -169,3 +169,10 @@ type Attributes struct {
 	// MD5 is an MD5 hash of the blob contents or nil if not available.
 	MD5 []byte
 }
+
+type SignedURLOption struct {
+	Method                   string
```

Comment:
what if the client set an invalid method?

Reply:
It's already handled by go.cloud. It will return error

https://github.com/google/go-cloud/blob/master/blob/blob.go#L1053